insights/specs/__init__.py

```diff
@@ -370,6 +370,8 @@ class Specs(SpecSet):
     ovirt_engine_console_log = RegistryPoint(filterable=True)
     ovirt_engine_server_log = RegistryPoint(filterable=True)
     ovirt_engine_ui_log = RegistryPoint(filterable=True)
+    ovs_appctl_fdb_show_bridge = RegistryPoint(multi_output=True)
+    ovs_vsctl_list_br = RegistryPoint()
```

Comment:
All things look good, but this `ovs_vsctl_list_br` is just a dependency datasource which is not necessary to put in the `\_\_init\_\_.py` file.